info/security.md

```diff
@@ -5,50 +5,16 @@ title: "Security"
 {% include JB/setup %}
 
 
-# [Start SSL](http://www.startssl.com/?app=0)
+# SSL/TLS Information
 
-Have to make sure your "client" certificate is renewed every year, so that you can register new domains.  I think you can only do this within two weeks of expiration.  It's a bit painful to use, but it is free.
+* [Let's Encrypt](https://letsencrypt.org/):  Freely available certificates for websites.
+Requires a client as the certficates are only valid for 90 days, but many clients
+exist.  I currently use [EFF Certbot](https://certbot.eff.org/)
+* [Start SSL](http://www.startssl.com/?app=0):  Also offers free certificates,
+but also several paid options.  Used to be a bit cumbersome to use, but unsure
```

Comment:
instead of "but unsure of their current status" would it be better to say something like "but may be improved after recent updates"?